pkg/kv/kvserver/kv_snapshot_strategy.go

```diff
@@ -0,0 +1,655 @@
+// Copyright 2025 The Cockroach Authors.
```

Comment:
I remember from somewhere that the copyright year should be copied from the old file when things move around.

Reply:
I remember this too. In this case, the IDE made the file - I'll pay more attention to it in the future.
I suspect it technically makes little difference, since the license no longer reverts to Apache-2.0 after some predetermined time.